src/ApplicationCore/Entities/FraudProtectionApiModels/AccountProtection/Signup.cs

```diff
@@ -34,5 +36,8 @@ public class EventMetadataAccountCreate : EventMetadata
         public string SignUpId { get; set; }
 
         public DateTimeOffset CustomerLocalDate { get; set; }
+
+        [JsonConverter(typeof(StringEnumConverter))]
+        public AssessmentType AssessmentType { get; set; }
```

Comment:
:-)

Reply:
yes, it was good to see this among the AP contract changes